proto/cosmos/gov/v1beta2/tx.proto

```diff
@@ -1,15 +1,15 @@
 syntax = "proto3";
-package cosmos.gov.v1beta1;
+package cosmos.gov.v1beta2;
```

Comment:
what about v1beta1 vote and deposit? I think we need to support them at least in a separate PR

Reply:
Why would we still want v1beta1 vote and deposit. I thought we'd want to bump all gov types together to v1beta2 (even if they weren't changed) so that you wouldn't have to import across multiple versions

Reply:
because it's needed for clients. changing the msg URL from v1beta1 -> v1beta2 means that keplr and other gov UIs will be broken